app/index.js

```diff
@@ -70,7 +72,18 @@ app.on('ready', () => {
 
     // config changes
     const cfgUnsubscribe = config.subscribe(() => {
+      const cfg_ = plugins.getDecoratedConfig();
```

Comment:
This is the new config, and is also temporary. This, I believe, follows the existing style for temporary variables using `_` at the end.